app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
+
+
+def create_person_list(people: list) -> list[Person]:
+    new_people_list = []
+
+    for human in people:
+        new_people_list.append(Person(human["name"], human["age"]))
```

Comment:
Consider using list comprehension to create the `new_people_list`. This will make the code more concise and is recommended by the checklist: 'When creating a list of instances from a collection (such as a list of dictionaries), it is recommended to use list comprehension rather than traditional for loops.'